attractions/dropdown/dropdown-shell.svelte

```diff
@@ -8,7 +8,7 @@
 
   export function toggle() {
     open = !open;
-    dispatch('change', open);
+    dispatch('change', { value: open});
```

Comment:
Don't forget a space on both sides of the braces

Reply:
I felt something was wrong but couldn't quite place it 😅